src/color_laws.jl

```diff
@@ -370,3 +370,63 @@ function f99_invum(x::Real, Rv::Real)
             optnir_axebv_y,
         )
 end
+
+# read and unpack tabulated data
+fpath = joinpath(dirname(pathof(DustExtinction)), "data/F19_tabulated.dat")
+data = readdlm(fpath, skipstart=1)
+const data_x, data_k, data_delta_k, data_sigma_k = (data[:, i] for i in 1:4)
```

Comment:
Defining `fpath` in this way will not work on Windows.  Also, you can avoid polluting the namespace with variables that you are not going to later use with a [`let`](https://docs.julialang.org/en/v1/base/base/#let):
```suggestion
const data_x, data_k, data_delta_k, data_sigma_k = let data = readdlm(joinpath(dirname(pathof(DustExtinction)), "data", "F19_tabulated.dat"), skipstart=1)
    (data[:, i] for i in 1:4)
end
```

Reply:
Oh, really cool! I'll read up more on `let`

Reply:
Also, completely forgot about the forward slash I left behind, thanks!